pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -89,7 +91,11 @@ private void setupSwagger() {
     beanConfig.setDescription("APIs for accessing Pinot broker information");
     beanConfig.setContact("https://github.com/apache/pinot");
     beanConfig.setVersion("1.0");
-    beanConfig.setSchemes(new String[]{CommonConstants.HTTP_PROTOCOL, CommonConstants.HTTPS_PROTOCOL});
+    if (CommonConstants.HTTPS_PROTOCOL.equalsIgnoreCase(_swaggerDefaultProtocol)) {
+      beanConfig.setSchemes(new String[]{CommonConstants.HTTPS_PROTOCOL, CommonConstants.HTTP_PROTOCOL});
```

Comment:
yeah, still needed at least for those quickstart examples, and perhaps for users not enabling TLS.

Reply:
oh, got your point. Taking the new configs suggested by you above, we ca save http here.

Reply:
hmm... on a second thought, I'd prefer to leave `http` there, in case someone wants to debug issues like when TLS doesn't work and he/she wants to switch to http for some quick checks.